src/main/java/org/jabref/gui/externalfiles/ImportHandler.java

```diff
@@ -128,8 +129,10 @@ protected List<ImportFilesResultItemViewModel> call() {
                             if (pdfImporterResult.hasWarnings()) {
                                 addResultToList(file, false, Localization.lang("Error reading PDF content: %0", pdfImporterResult.getErrorMessage()));
                             }
-
                             if (!pdfEntriesInFile.isEmpty()) {
+                                for (BibEntry entry : pdfEntriesInFile) {
```

Comment:
That's stupid. We have a method which takes of all of those things. Try to understand the code first before doing any changes 
see addResultToList and it takes a Path as argument.
You basically just have to call FileUtil.relative before adding it tot he list

Reply:
Hi @Siedlerchr ,

Thanks for your review. Please help me to clear my doubts
as per my understanding,

1. The addResultToList() methiod is just used for logging purpose, as we add the result of the operations and just use it to log at the end of the operation.

2. As per your suggestion, I am already using the FileUtil.relativize() method, this call is in the private relativize() method that has been added to the ImportHandler class. This code was approved by @koppor in the comment section of the issue, and we have created the pull request for the same.

If you suggest, instead of having a different private relativize() method in the class, directly implement it in the following way,

for (BibEntry entry : pdfEntriesInFile) {
        entry.setField(StandardField.FILE, FileUtil.relativize(file));
    }
    
I can implement it in this way as well. As of now we went ahead with the approach discussed and approved in the issue comments section. Please let us know your thoughts and feel free to correct me if my understanding is wrong.

Thanks,
Arshad

Reply:
@Arshadpd Any feedback on this?

Reply:
Form my point of view the current implementaton is fine and we can leave the more complex one as a future task

Reply:
The current implementation removes all links contained in the BibEntry objects contained. Thus, if more than one file link is contained, it is removed. I need to recheck, if this case really happens. Reading my comment from last week (https://github.com/JabRef/jabref/pull/11409#discussion_r1650954259), it might happen.

Moreover, I had code comments even if we agree that (potentially) destroying data is OK. These should be addressed in all cases. -- Maybe, it is OK to create a log output (warn) if file entries are overwritten.

We spend now at least one hour for reviewing and we should make use of the intellecutal propery we crafted there.